src/components/DocumentView/BlockContentRef.tsx

```diff
@@ -1,27 +1,77 @@
-import { DocumentBlockContentRef } from '@gitbook/api';
+import { CustomizationThemedURL, DocumentBlockContentRef } from '@gitbook/api';
 
 import { Card, Emoji } from '@/components/primitives';
+import { getSpaceCustomization, ignoreAPIError } from '@/lib/api';
+import { ResolvedContentRef } from '@/lib/references';
 
 import { BlockProps } from './Block';
+import { Image } from '../utils';
+
 
 export async function BlockContentRef(props: BlockProps<DocumentBlockContentRef>) {
     const { block, context, style } = props;
-    const kind = block?.data?.ref?.kind;
 
     const resolved = await context.resolveContentRef(block.data.ref, {
         resolveAnchorText: true,
     });
+
     if (!resolved) {
         return null;
     }
 
+    const kind = block?.data?.ref?.kind;
+    if (resolved.active && kind === 'space') {
+        return <SpaceRefCard {...props} resolved={resolved} />;
+    }
+
+    return (
+        <Card
+            leadingIcon={resolved.emoji ? <Emoji code={resolved.emoji} style="text-xl" /> : null}
+            href={resolved.href}
+            title={resolved.text}
+            style={style}
+        />
+    );
+}
+
+async function SpaceRefCard(props: { resolved: ResolvedContentRef } & BlockProps<DocumentBlockContentRef>) {
+    const { context, style, resolved } = props;
+    const spaceId = context.contentRefContext?.space.id;
+
+    if (!spaceId) { return null; }
+
+    const spaceCustomization = await ignoreAPIError(getSpaceCustomization(spaceId));
+    const customFavicon = spaceCustomization?.favicon;
+    const customEmoji = customFavicon && 'emoji' in customFavicon ? customFavicon.emoji : null;
+    const customIcon = customFavicon && 'icon' in customFavicon ? customFavicon.icon : null;
+
     return (
         <Card
-            leadingIcon={resolved.emoji ? <Emoji code={resolved.emoji} /> : null}
+            leadingIcon={customIcon ? <BlockContentRefIcon icon={customIcon} /> : customEmoji ? <Emoji code={customEmoji} style="text-xl" /> : null}
             href={resolved.href}
-            preTitle={kind}
             title={resolved.text}
             style={style}
         />
     );
 }
+
+function BlockContentRefIcon(props: { icon: CustomizationThemedURL }) {
+    const { icon } = props;
+            
+    return icon ? <Image 
+        priority='lazy' 
+        alt=""
```

Comment:
```suggestion
        alt={`Logo for ${resolved.text}`}
```

Let's set an alt text

Reply:
I'm not sure that is better. 

An empty alt text tends to be the way to handle decorative images, in other words, images that don't provide any useful information. They get ignored by a screen reader.  Similar example from [W3C](https://www.w3.org/WAI/tutorials/images/decorative/#example-2-decorative-image-as-part-of-a-text-link).

Knowing that there is a logo there doesn't really have any value if you can't see the logo. Unless we describe the logo it's just adding noise.

Reply:
Okay, I did see that in `HeaderLogo` we use an alt text to maybe worth removing that too.